src/scripts/check-definitions.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python
+# Licensed to the .NET Foundation under one or more agreements.
+# The .NET Foundation licenses this file to you under the MIT license.
+# See the LICENSE file in the project root for more information.
+#
+# check-definitions.py
+#  This script checks the consistency between compiler definitions
+# of the native part of CoreCLR and managed part (mscorlib.dll) of
+# CoreCLR
+#
+# Usage:
+#   $ ./check-definitions.py [Definition File 1] [String of definitions]
+#
+#      Definitions in a definition string should be seperated by a
+#     semicolon.
+#
+# (c) 2016 MyungJoo Ham <myungjoo.ham@samsung.com>
+
+import sys
+import re
+
+debug = 0
+
+# For the native part, return the sorted definition array.
+def loadDefinitionFile(filename):
+    result = []
+    f = open(filename, 'r')
+    for line in f:
+        theLine = line.rstrip("\r\n").strip()
+	if (len(theLine) > 0):
```

Comment:
I'll fix it and resubmit the commits; I might need to make .vimrc to detect the type of repositories (whether it's Linux kernel or not. I have other repositories in .C that uses spaces, not tabs, either). :(